app/models/user.rb

```diff
@@ -19,6 +21,18 @@ def self.new_token
     SecureRandom.urlsafe_base64
   end
 
+  def self.create_anonimous
+    anonimous_token = SecureRandom.hex(8)
+    create username: anonimous_token, email: "#{anonimous_token}@stub",
```

Comment:
I changed `create` to `create!` and still have no idea what am I doing. I hardcoded valid parameters there, it's not much flexible but always valid. Having error if something goes wrong it's a good thing, of course, but what I should do with it? What "get a cryptic "method not found"" means and where is somewhere down the flow?